pallets/loans-ref/src/lib.rs

```diff
@@ -17,14 +17,14 @@
 //!
 //! The following actions are performed over loans:
 //!
-//! | Extrinsics                         | Role      |
-//! |------------------------------------|-----------|
-//! | [`Pallet::create()`]               | Borrower  |
-//! | [`Pallet::borrow()`]               | Borrower  |
-//! | [`Pallet::repay()`]                | Borrower  |
-//! | [`Pallet::write_off()`]            | Any       |
-//! | [`Pallet::admin_write_off()`]      | LoanAdmin |
-//! | [`Pallet::close()`]                | Borrower  |
+//! | Extrinsics                    | Role      |
+//! |-------------------------------|-----------|
+//! | [`Pallet::create()`]          | Borrower  |
+//! | [`Pallet::borrow()`]          | Borrower  |
+//! | [`Pallet::repay()`]           | Borrower  |
+//! | [`Pallet::write_off()`]       | Any       |
```

Comment:
Super minor nit, but: to be consistent with `Pallet::apply_change()`, I would remove `Any` here

Reply:
Makes sense, done!